CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 3.5)
-project(geometric_shapes VERSION 1.0.0)
+project(geometric_shapes)
+
+# Set VERSION from package.xml
+find_package(ament_cmake_core REQUIRED)
```

Comment:
You are going to `find_package(ament_cmake)` further below. Would that already include core?
Maybe move `ament_package_xml()` further down then?

Reply:
`ament_cmake` works

Reply:
I adapted the suggestion appropriately in the main PR https://github.com/ros-planning/geometric_shapes/pull/162